docs/content/users/quickstart.md

```diff
@@ -633,38 +634,27 @@ The Laravel project type can be used for [Statamic](https://statamic.com/) like
 
 ```bash
 mkdir my-sulu-site && cd my-sulu-site
-ddev config --project-type=php --docroot=public --upload-dirs=uploads --database=mysql:8.0
+ddev config --project-type=php --docroot=public --upload-dirs=uploads --database=mysql:8.0 --webimage-extra-packages="xmlstarlet"
 ddev start
 ddev composer create sulu/skeleton
-```
-
-Create your default webspace configuration `mv config/webspaces/example.xml config/webspaces/my-sulu-site.xml` and adjust the values for `<name>` and `<key>` so that they are matching your project:
-
-```bash
-<?xml version="1.0" encoding="utf-8"?>
-<webspace xmlns="http://schemas.sulu.io/webspace/webspace"
-          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-          xsi:schemaLocation="http://schemas.sulu.io/webspace/webspace http://schemas.sulu.io/webspace/webspace-1.1.xsd">
-    <!-- See: http://docs.sulu.io/en/latest/book/webspaces.html how to configure your webspace-->
-
-    <name>My Sulu CMS</name>
-    <key>my-sulu-cms</key>
+# Create your default webspace configuration `config/webspaces/my-sulu-site.xml`
+# from `config/webspaces/website.xml`. The command below will adjust the values
+# for `<name>` and `<key>`, so that they are matching your project:
+# <name>My Sulu Site</name>
+# <key>my-sulu-site</key>
+ddev exec 'name="$(echo "${DDEV_PROJECT}" | sed "s/\b\(.\)/\U\1/g" | tr "-" " ")"; key="${DDEV_PROJECT}"; xmlstarlet ed -u "//_:webspace/_:name" -v "${name}" -u "//_:webspace/_:key" -v "${key}" -u "//_:portals/_:portal/_:name" -v "${name}" -u "//_:portals/_:portal/_:key" -v "${key}" config/webspaces/website.xml > config/webspaces/${DDEV_PROJECT}.xml && rm -f config/webspaces/website.xml'
```

Comment:
(I think this is OK with the modest update above, saying that they can edit themselves or use this magic command. Still a bit uncomfortable with it.) 

I understand that you're trying to simplify the technique here, but I'm not actually convinced that people will understand or be able to recreate what we're doing here. I may prefer the old technique, which has clear docs. We want people to be able to clearly understand every step, and perhaps with this approach we're not doing that.

Reply:
I'll replace it with simpler `ddev exec sed ...` approach.